rules/aws_iam_group_policy_too_long.go

```diff
@@ -0,0 +1,65 @@
+package rules
+
+import (
+	"fmt"
+	hcl "github.com/hashicorp/hcl/v2"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+	"github.com/terraform-linters/tflint-ruleset-aws/project"
+	"regexp"
+)
+
+// TODO: Write the rule's description here
+// AwsIAMGroupPolicyTooLongRule checks that the policy length is less than 5,120 characters
+type AwsIAMGroupPolicyTooLongRule struct {
+	resourceType  string
+	attributeName string
+}
+
+// NewAwsIAMGroupPolicyTooLongRule returns new rule with default attributes
+func NewAwsIAMGroupPolicyTooLongRule() *AwsIAMGroupPolicyTooLongRule {
+	return &AwsIAMGroupPolicyTooLongRule{
+		resourceType:  "aws_iam_group_policy",
+		attributeName: "policy",
+	}
+}
+
+// Name returns the rule name
+func (r *AwsIAMGroupPolicyTooLongRule) Name() string {
+	return "aws_iam_group_policy_too_long"
+}
+
+// Enabled returns whether the rule is enabled by default
+func (r *AwsIAMGroupPolicyTooLongRule) Enabled() bool {
+	return true
+}
+
+// Severity returns the rule severity
+func (r *AwsIAMGroupPolicyTooLongRule) Severity() string {
+	return tflint.ERROR
+}
+
+// Link returns the rule reference link
+func (r *AwsIAMGroupPolicyTooLongRule) Link() string {
+	return project.ReferenceLink(r.Name())
+}
+
+// TODO: Write the details of the inspection
+// Check checks the length of the policy
+func (r *AwsIAMGroupPolicyTooLongRule) Check(runner tflint.Runner) error {
+	return runner.WalkResourceAttributes(r.resourceType, r.attributeName, func(attribute *hcl.Attribute) error {
+		var policy string
+		err := runner.EvaluateExpr(attribute.Expr, &policy, nil)
+		whitespaceRegex := regexp.MustCompile(`\s+`)
```

Comment:
Go's regexp compile is slow, so I prefer to compile the regexp in `NewAwsIAMGroupPolicyTooLongRule` rather than the `Check` function, which runs over and over again.
https://github.com/terraform-linters/tflint-ruleset-aws/blob/20a14646ec7402659b997ee70858e59968af25cf/rules/models/aws_acm_certificate_invalid_certificate_body.go#L30

The good point about this way is that if you have a problem with the regexp, you will get a panic early and notice the problem (`NewAwsIAMGroupPolicyTooLongRule` will be always called).